Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/CommonSettings.java

```diff
@@ -152,22 +154,25 @@ private static void setEnvironment(@NotNull String env, Map<String, String> endP
     }
 
     private static boolean isUsingDeprecatedBaseFolder(String baseDir, String deprecated) {
-        return !FileUtil.isNonEmptyFolder(baseDir) && FileUtil.isNonEmptyFolder(deprecated);
+        File baseFile = new File(baseDir);
+        return !baseFile.exists() && FileUtil.isNonEmptyFolder(deprecated);
     }
 
     private static void initBaseDir(@NotNull String basePath) throws IOException {
         File baseDir = new File(basePath);
         if(!baseDir.exists()){
             FileUtils.forceMkdir(baseDir);
         }
-        Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        if (Utils.isWindows()) {
+            Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        }
     }
 
-    private static void copyResourcesFromDeprecatedFolder(String baseDir, String deprecatedDir) {
+    private static void moveResourcesFromDeprecatedFolder(String baseDir, String deprecatedDir) {
         try {
-            FileUtils.copyDirectory(new File(deprecatedDir), new File(baseDir));
+            FileUtils.moveDirectory(new File(deprecatedDir), new File(baseDir));
```

Comment:
we should only move our files like AadProvider.json, not the entire folder.

Reply:
Fixed